documentation/src/docs/asciidoc/release-notes/release-notes-5.7.0-M1.adoc

```diff
@@ -15,7 +15,7 @@ on GitHub.
 
 ==== Bug Fixes
 
-* ❓
+* The javadoc of `TestTemplateInvocationContextProvider` has been fixed: it's allowed to return an empty stream. The error message when all provided streams are empty is now more helpful.
```

Comment:
This should be in the `JUnit Jupiter` section.
Also, can we call it a bug fix really? Or is it just an improvement? 🤔